install_config/aggregate_logging_sizing.adoc

```diff
@@ -192,6 +192,20 @@ journal file corruption.
 link:https://rhn.redhat.com/errata/RHBA-2016-1026.html[This issue is resolved in
 later versions of systemd].
 
+*Fluentd*
+
+As of {product-title} 3.7, OpenShift tests its logging architecture with 1KB log
+messages, approximately 1000 messages/second. This is the upper limit
+of what the infrastructure can handle.
+
+For more stability, sending similarly sized logs, at a rate of 800-900
+messages/second on a consistent basis, is recommended.
+
+The limiting factor in scaling the logging infrastructure to a higher throughput
+resides with Fluentd (the log aggregation solution). Currently, the way Fluentd
+operates, the number of threads that can process the logs
+collected is limited, and as a result, have the defined upper bounds. 
+
```

Comment:
w.r.t. @richm's comment, I suggest replacing the final paragraph with:

Scaling the logging infrastructure to a higher throughput is currently limited by Fluentd 0.12, Ruby 2.0, OpenShift's configuration of Fluentd and the log processing code used in OpenShift.

If that is too geeky, maybe something like "limited by the current versions of the OpenShift logging components and their configuration"